.buildkite/scripts/run_e2e_tests.sh

```diff
@@ -44,8 +44,9 @@ build_logstash
 build_plugin
 
 ###
-# TODO: disable ls command and enable with Python E2E scripts
-ls -a
-# Install prerequisites and run E2E tests
-#pip install -r .buildkite/scripts/e2e/requirements.txt
-#python3 .buildkite/scripts/e2e/main.py
\ No newline at end of file
+# Install pip and E2E prerequisites, then run E2E tests
+sudo apt install -y python3-pip
+pip --version
```

Comment:
@dliappis it seems `pip` is not available in the VM. I am trying to install it here but facing a prompt blocker. I wonder if we can make built-in pip or overcome prompt. 
<img width="1112" alt="image" src="https://github.com/elastic/logstash-filter-elastic_integration/assets/99575341/283a7862-0eb0-4e43-9840-dba9262039e0">

Reply:
The right command would be `pip3` not `pip`. However, if I were you I'd completely stop using the Python shipped by the Ubuntu 22.04 distro (and thus not `apt install python3-pip` either), but instead leverage one of the pre-installed Python 3 versions via pyenv using a virtualenv, which is the standard way to obtain isolation in the Python ecosystem.

Practically, in your script you'd simply need to specify which version you want to use (e.g. using `pyenv local 3.10.13` which creates a `.python-version` file with this version) and then create a virtualenv using `python3 -mvenv .venv` and activate this using `source .venv/bin/activate`. Note that a standard practice is to commit the `.python-version` in your repo, similar to how we commit `.ruby-version` in Logstash related repos.

With the above steps done, you can use pip with `python3 -mpip ...`. Example below.

```
buildkite-agent@dl-logstash-ubuntu-2204:~$ pyenv versions
* system (set by /opt/buildkite-agent/.pyenv/version)
  3.6.13
  3.10.13
  3.11.5
buildkite-agent@dl-logstash-ubuntu-2204:~$ pyenv local 3.10.13
buildkite-agent@dl-logstash-ubuntu-2204:~$ cat .python-version 
3.10.13
buildkite-agent@dl-logstash-ubuntu-2204:~$ pyenv version
3.10.13 (set by /opt/buildkite-agent/.python-version)
buildkite-agent@dl-logstash-ubuntu-2204:~$ python3 -mvenv .venv
buildkite-agent@dl-logstash-ubuntu-2204:~$ source .venv/bin/activate
(.venv) buildkite-agent@dl-logstash-ubuntu-2204:~$ python3 -mpip install urllib3
Collecting urllib3
  Downloading urllib3-2.2.1-py3-none-any.whl (121 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 121.1/121.1 kB 4.3 MB/s eta 0:00:00
Installing collected packages: urllib3
Successfully installed urllib3-2.2.1

[notice] A new release of pip is available: 23.0.1 -> 24.0
[notice] To update, run: pip install --upgrade pip
```